app/components/LiveDock.vue.ts

```diff
@@ -188,11 +188,8 @@ export default class LiveDock extends Vue {
     this.chatService.refreshChat();
   }
 
-  get hideChat() {
-    // previewEnabled is poorly named.  It really just means stuff
-    // we disable while resizing.  It should not be in the customization
-    // service.  This is lazy.
```

Comment:
I thought of creating a new service, but decided we should keep it in the customization service for now.  It's not technically a user setting, but it's related to resizing, which is handled by the customization service.  So it's not the worst place for now.